### PR TITLE
Standardize README across raw BIDS subdatasets

### PR DESCRIPTION
**Pull request by bpinsard**
Started to write a template for basic infos (and mainly links to centralized doc) that would be available in the BIDS readme.
What else should we included here?
- Authors?
- Contacts?
- Link to DTA (docs).
- License?

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
 
+This BIDS dataset is part of the [Courtois-Neuromod dataset](https://docs.cneuromod.ca/).
+
+This subdataset contains functional MRI data collected with xxxxx task.
+
+It contains:
```
Comment (lunebellec) [inline README]: suggestion: merge "This subdataset contains (...)" with "It contains".

File: README [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+- raw neuroimaging data in BIDS format
+- behavioral variables
+- physiological data
```
Comment (lunebellec) [inline README]: Suggestion: add author and access information. 

This dataset was designed by XX, with input from XX, XX. The subdataset was collected  by XX, XX. Further improvements on data organization as well as quality control have been implemented by XX, XX. This subdataset also benefited from resources assembled by the entire [Courtois NeuroMod team](https://docs.cneuromod.ca/en/latest/AUTHORS.html).

Courtois NeuroMod data are freely shared with the scientific community to advance research at the interface of neuroscience and artificial intelligence. Access to to the data is based on a registered access model, see the [Courtois NeuroMod website](https://docs.cneuromod.ca/en/latest/ACCESS.html) for more information. Redistribution of data outside of the Courtois NeuroMod data bank is prohibited.

File: README [hunk 3]
```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
```
Comment (lunebellec) [inline README]: We are using two abbreviations: cneuromod (short) and Courtois NeuroMod (long). I suggest to stick to Courtois NeuroMod here.

File: README [hunk 4]
```diff
@@ -1,2 +1,10 @@
...
 
+This BIDS dataset is part of the [Courtois-Neuromod dataset](https://docs.cneuromod.ca/).
```
Comment (lunebellec) [inline README]: See my comment above Courtois-Neuromod -> Courtois NeuroMod

Comment (lunebellec) [reply README]: I would also remove the reference to BIDS from this sentence. Some of our subdataset will be BIDS-like, but not strictly BIDS compliant.

File: README [hunk 5]
```diff
@@ -1,2 +1,10 @@
...
+
```
Comment (lunebellec) [inline README:5]: I would add a sentence here on BIDS. 

OPTIONAL: 

This subdataset is organized following the [brain imaging data structure](https://bids.neuroimaging.io/).
OR
This subdataset is organized following principles which mimic the [brain imaging data structure](https://bids.neuroimaging.io/), but it does not conform to an established standard.

File: README [hunk 6]
```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
```
Comment (lunebellec) [inline README:3]: I would add a general intro to the project:

The Courtois project on Neural Modelling ([Courtois NeuroMod](https://cneuromod.ca)) aims at training artificial neural networks using extensive experimental data on individual human brain activity and behaviour.

**Review by lunebellec — CHANGES_REQUESTED** [review]

**Comment by lunebellec** [comment]
Thanks for getting this started Basile! 

I am not sure what to do with the failed bids validator test. Could we add some minimal common folder structure so that the test passes?

**Comment by bpinsard** [comment]
> Thanks for getting this started Basile!

Thanks for your helpful comments. 
> I am not sure what to do with the failed bids validator test. Could we add some minimal common folder structure so that the test passes?

Creating a fake BIDS would require removing it when applying the template. 
I guess we could just add a condition that would skip it on this template (by detecting the git url), will look into that option.

**Comment by bpinsard** [comment]
@pbellec @julieaboyle1 are we ok with the latest version. I would like to apply that README to all existing datasets for the release. Thanks

**Review by lunebellec — APPROVED** [review]

**Comment by lunebellec** [comment]
Looking great! Thanks @bpinsard !

**Pull request merged**